### ISSUE TITLE
Rucio tracers do not read old messages when restarted

### ISSUE THREAD

**Issue by dynamic-entropy** [dmwm/CMSRucio#541]
The rucio-tracers index on Opensearch is empty. 
https://monit-opensearch.cern.ch/dashboards/goto/2365b4941590285fbabdf4d73d44ad0a?security_tenant=global

The traces pods show "connection refused".

**Comment by dynamic-entropy** [comment]
Hi @ericvaandering 

The traces are back in OpenSearch after the restart. However, are we not reading this from some queue? 
From my understanding of them, the data should remain in the queue until consumed, which does not seem to be the case. 

Can someone clarify?

**Comment by ericvaandering** [comment]
The Kronos daemon is reading from the queue.

**Comment by dynamic-entropy** [comment]
Hi @yuyiguo 
Do you know what is the retention policy and capacity of the 3 input queues to rucio-tracers?
And similarly acknowledgement mode of our consumers? 

We need to prepare for such future outages of the tracer as a first step to ensuring almost all tracers are captured.

**Comment by yuyiguo** [comment]
Rucio tracers are part of the consumers.  If you want to know the queues in detail, you need to contact the data producers. This is a way for Rucio to collect the data, but we are not the producers and cannot control the data.

**Comment by dynamic-entropy** [comment]
Hi Yuyi, I am not asking about the data (and thus it has nothing to do with having the knowledge of the producer), but the "queue" that we (rucio-tracers) are consuming and the producers are publishing to.
I think it is vital for us to know the retention of the queues to know the max time we can tolerate for rucio-tracers being down, without missing data.

And similarly, producers have nothing to do with the acknowledgement mode of the consumers.

**Comment by yuyiguo** [comment]
I tried to explain to you about the queues used by Rucio tracer. Let me try again. The queues were there long before Rucio started to use them. People created these queues for different purposes that I don't know. While we started to collect the traces, the existing queues were used because there were no direct ways to get the data into the Rucio system.  
I did not set up the queues, but I knew AMQ admin will not let the queues grow too big. If you want to know how the queues are configured, talk to the producers or the AMQ admin.  They are the persons setting up these.
